lxd/sys/cgroup.go

```diff
@@ -29,4 +30,5 @@ var cGroups = []struct {
 	warn string
 }{
 	{"blkio/", cGroupMissing("blkio", "I/O limits will be ignored")},
+	{"cpu/", cGroupMissing("CPU controller", "CPU time limits will be ignored")},
```

Comment:
Just a small nit but having the trailing slash there after the name looks a bit odd.
Maybe change "path" to "name" in the struct and do away with the trailing slash?

Reply:
Oh, I see with 4c43f12d874869fa1faf789f08b892c67b179854 you use that to check for a particular file being present so that won't work. You could still drop the trailing slashes though :)

Reply:
Ok, I dropped the trailing slash in 3ff796a6d7d5143528025c13d79be8b165c81abe.